src/plugins/expressions/common/expression_types/specs/datatable.ts

```diff
@@ -42,19 +49,31 @@ export type DatatableColumnType = 'string' | 'number' | 'boolean' | 'date' | 'nu
  */
 export type DatatableRow = Record<string, any>;
 
+export interface DatatableColumnMeta {
+  type: DatatableColumnType;
+  field?: string;
+  params?: SerializableState | {};
```

Comment:
Why the `| {}`...? An empty object is still a `SerializableState`...?

Reply:
@lukeelmers any idea ?

Reply:
I don't think this is required, the current typings for `SerializableState` should work with an empty object.

TBH I can't exactly remember why I ended up putting this in the AggConfig typings.

Reply:
Another question: Do we want `params` to be optional? Or should they be required and just an empty object if there's no value?

I guess optional makes sense if `field` is also optional.

Reply:
yeah i think optional makes sense